### PR TITLE
Update stockpile tech descriptions

### PR DESCRIPTION
**Pull request by agrrr3**
starting to update tech descriptions. should  solve (partly) #2080

Edited to move "partly" between the 'solve' and the issue number, just to be safe against prematurely closing the issue (I don't see 'solve' on the list of autoclosing keywords, but better safe than sorry)

### CODE REVIEW

File: default/stringtables/en.txt [hunk 1]
```diff
@@ -10498,7 +10498,7 @@ PRO_INTERSTELLAR_ENTANGLEMENT_FACTORY_DESC
 '''Quantum entanglement of factory control rooms allows to operate machines remotely from other star systems.
```
Comment (Dilvish-fo) [inline default/stringtables/en.txt]: "...allows the operation of machines..."

File: default/scripting/techs/production/INTERSTELLAR_ENTANGLEMENT_FACTORY.focs.txt [hunk 2]
```diff
@@ -19,6 +19,7 @@ Tech
                 OwnedBy empire = Source.Owner
                 Species
             ]
+            accountinglabel = "INTERSTELLAR_ENTANGLEMENT_FACTORY_POPULATION_BONUS_LABEL"
```
Comment (Dilvish-fo) [inline default/scripting/techs/production/INTERSTELLAR_ENTANGLEMENT_FACTORY.focs.txt:22]: This stringtable entry needs to be provided.

File: default/scripting/techs/production/INTERSTELLAR_ENTANGLEMENT_FACTORY.focs.txt [hunk 3]
```diff
@@ -27,6 +28,7 @@ Tech
                 OwnedBy empire = Source.Owner
                 Focus type = "FOCUS_STOCKPILE"
             ]
+            accountinglabel = "INTERSTELLAR_ENTANGLEMENT_FACTORY_FIXED_BONUS_LABEL"
```
Comment (Dilvish-fo) [inline default/scripting/techs/production/INTERSTELLAR_ENTANGLEMENT_FACTORY.focs.txt:31]: This stringtable entry is likewise missing.

**Review by Dilvish-fo — DISMISSED** [review]
Additionally, on lines not closely enough involved here for me to be able to add comments directly to them under this PR, we are still missing the stringtable entries for BLD_STOCKPILING_CENTER, BLD_STOCKPILING_CENTER_LABEL, and for the accounting labels for Generic Supplies

**Comment by Vezzra** [comment]
> we are still missing the stringtable entries for BLD_STOCKPILING_CENTER, BLD_STOCKPILING_CENTER_LABEL, and for the accounting labels for Generic Supplies

Well, @agrrr3 did state that this PR only partly addresses #2080.

**Comment by Dilvish-fo** [comment]
> Well, @agrrr3 did state that this PR only partly addresses #2080.

Yes, I noticed that after I posted, but I think the comments are still useful signposts for him or whoever else finishes up the work.  And I recall that he is quite busy preparing for, or caring for, a new baby.  So let me note, @agrrr3 I appreciate you having taken the time to get this initial PR up, it is certainly a helpful start, and if you need to pass the baton just say so.

File: default/stringtables/en.txt [hunk 4]
```diff
@@ -10498,7 +10498,7 @@ PRO_INTERSTELLAR_ENTANGLEMENT_FACTORY_DESC
...
 Decoupling workforce and the means of production helps to produce just in time and increases efficiency of the imperial stockpile service.
 
-This is an upgrade technology for the imperial stockpile, which increases the [[metertype METER_STOCKPILE]] of stockpiling-focused planets by 3.'''
+This is an upgrade technology for the imperial stockpile. Two effects allow drawing more PP from the stockpile: Increases the [[metertype METER_STOCKPILE]] by 6. Increases it by 0.02 per population.'''
```
[the remark this answers is not in the document]
Comment (agrrr3) [reply default/stringtables/en.txt]: yes this was intended to mirror the differences between the different levels of species bonus. 

regarding precedent for bonus plus unlock. i think there was one recent tech change where something similar was added, but it is definitly mixing up categories.

**Comment by agrrr3** [comment]
Hi peeps, i repushed with the requested changes (and some i did while i was offline). What is not addressed: we are still missing the stringtable entries for BLD_STOCKPILING_CENTER, BLD_STOCKPILING_CENTER_LABEL, and for the accounting labels for Generic Supplies, INTERSTELLAR_ENTANGLEMENT_FACTORY_POPULATION_BONUS_LABEL, INTERSTELLAR_ENTANGLEMENT_FACTORY_FIXED_BONUS_LABEL.
I wanted actually to streamline the effect labels for the techs but didnt get around to it. So generally somebody should do a round working on the accounting labels. 

Im travelling now and wont have more time to put into freeorion for a while, so somebody should take over. Thanks for the collaboration, i'm out ;)

**Comment by Vezzra** [comment]
@Dilvish-fo, has @agrrr3 addressed all your requests (aside from the still missing entries of course)? It would be good if we could merge this before the creation of the release branch, spares us the cherry picking.

**Pull request merged**